src/Microsoft.FeatureManagement/FeatureVariantAssignmentContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+namespace Microsoft.FeatureManagement
+{
+    /// <summary>
+    /// Contextual information needed during the process of feature variant assignment
+    /// </summary>
+    public class FeatureVariantAssignmentContext
```

Comment:
>FeatureVariantAssignmentContext [](http://example.com/codeflow?start=17&length=31)

Are we planning to add more members to this class? If not, can we directly use `FeatureDefinition` instead of `FeatureVariantAssignmentContext`?

Reply:
There isn't currently a plan, but it's not impossible. Right now the context is here to be symmetric with IFeatureFilter. Both IFeatureFilter and IFeatureVariant assigner benefit from taking a `Context` type because if we do decide to add anything to the context we don't have to break the interface.